app/services/chatbot/chatbot-interfaces.ts

```diff
@@ -1,141 +1,170 @@
 // state
-export interface ChatbotApiServiceState {
+export interface IChatbotApiServiceState {
```

Comment:
We extract interfaces into a separated file to reach 2 goals:
1) The file contains a restricted API that is safe to use for child-window, tests, and remote API clients. Restricted API can provide only serializable types.
2) The file name convention is `service-name-api.ts`. That helps to documentation generator publish only restricted API.

If you have questions pls DM me.